src/allotropy/parsers/perkin_elmer_envision/perkin_elmer_envision_structure.py

```diff
@@ -55,61 +53,51 @@ class PlateInfo:
     chamber_temperature_at_start: float | None
 
     @staticmethod
-    def get_series(reader: CsvReader) -> pd.Series[str]:
+    def get_series(reader: CsvReader) -> SeriesData:
         assert_not_none(
             reader.pop_if_match("^Plate information"),
             msg="Unable to find expected plate information",
         )
 
-        return df_to_series(
-            assert_not_none(
-                reader.pop_csv_block_as_df(),
-                "Plate information CSV block",
-            )
-        ).replace(np.nan, None)
-
-    @staticmethod
-    def get_plate_number(series: pd.Series[str]) -> str:
-        return try_str_from_series(
-            series,
-            "Plate",
-            msg="Unable to find plate number",
+        return SeriesData(
+            df_to_series(
+                assert_not_none(
+                    reader.pop_csv_block_as_df(),
+                    "Plate information CSV block",
+                )
+            ).replace(np.nan, None)
         )
 
-    @staticmethod
-    def get_barcode(series: pd.Series[str], plate_number: str) -> str:
```

Comment:
is this not useful anymore as a standalone function? I guess it's only called once?

Reply:
Yeah, it's called twice, but it's smaller now, and, I plan to dedup the two call sites in a followup.